.coveragerc

```diff
@@ -9,6 +9,7 @@ omit =
     invoicing_app/admin.py
     invoicing_app/models.py
     invoicing_app/management/commands/generate_tax_receipts_old.py
+    invoicing_app/management/commands/update_incomplete_tax_receipts.py
```

Comment:
If you are going to test this in the next sprint, you shouldn't add it to the omit list. Coverage report should reflect that there is something that needs to be tested.